google-cloud-core/pom.xml

```diff
@@ -63,6 +63,11 @@
       </exclusions>
     </dependency>
     <dependency>
+      <groupId>com.google.appengine</groupId>
+      <artifactId>appengine-api-1.0-sdk</artifactId>
+      <version>1.9.50</version>
+    </dependency>
```

Comment:
The code is set up like it is so that we can avoid adding this appengine dependency. We need to keep doing this with reflection.